module_utils/network_lsr/argument_validator.py

```diff
@@ -1611,7 +1645,9 @@ def _validate_post(self, value, name, result):
                         "invalid 'interface_name' '%s'" % (result["interface_name"]),
                     )
             else:
-                if not result.get("mac"):
+                if not result.get("mac") and (
+                    not result.get("match") or {"path": []} == result.get("match")
```

Comment:
Please put the expected values on the right-hand side. This can also simplified to avoid the expected value:

```suggestion
                if not result.get("mac") and (
                    not result.get("match") or not result.get("match", {}).get("path")
```

Reply:
Updated.